xrpl/models/currencies/currency.py

```diff
@@ -7,6 +7,7 @@
 from typing import Union
 
 from xrpl.models.currencies.issued_currency import IssuedCurrency
+from xrpl.models.currencies.mpt_currency import MPTCurrency
 from xrpl.models.currencies.xrp import XRP
 
-Currency = Union[IssuedCurrency, XRP]
+Currency = Union[IssuedCurrency, MPTCurrency, XRP]
```

Comment:
Do all XRPL transactions and requests allow `MPTCurrency` ? This `Currency` type is used in [book_offers](https://github.com/XRPLF/xrpl-py/blob/109c413088e24044c3895e37e1c526b8b26d3d16/xrpl/models/requests/book_offers.py#L23) and [amm_info](https://github.com/XRPLF/xrpl-py/blob/109c413088e24044c3895e37e1c526b8b26d3d16/xrpl/models/requests/amm_info.py#L26) RPCs.

Users could mis-interpret the type hints.

Reply:
Those RPCs can be updated, but honestly I don't think that's really worth doing because MPT support is coming for those RPCs soon enough anyways.

Reply:
alright